app/src/main/res/values/strings.xml

```diff
@@ -101,6 +101,8 @@
     <string name="about_summary_authors">"© 2016–2021 Tobias Zwick and contributors"</string>
     <string name="about_title_repository">"Source Repository"</string>
     <string name="about_summary_repository">"on GitHub"</string>
+    <string name="about_title_faq">"Frequently Asked Questions"</string>
+    <string name="about_summary_faq">"Please read these FAQ first if you have issues."</string>
```

Comment:
Should be removed. It should be clear what a FAQ is, the summary text is also too long

Reply:
shortened; is it better now?